content/500-platform/550-platfrom-cli/commands.mdx

```diff
@@ -0,0 +1,230 @@
+---
+title: 'Commands'
+metaTitle: 'Platform CLI: Commands'
+earlyaccess: true
+tocDepth: 5
+---
+
+<TopBlock>
+
+This document describes the Prisma Data Platform's integrated Prisma CLI commands, arguments, and options.
+
+</TopBlock>
+
+## Getting started
+
+To get started, ensure you have the [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) updated to version `5.7.0` or later. This is necessary to access the Platform through the Prisma CLI.
```

Comment:
what does Prisma Client version number have to do with the CLI? should this not rather refer to the Prisma CLI instead of Prisma Client?

Reply:
This is mentioned in the original doc, I simply transferred this section of the content from there. Nice catch 💯 !